src/commands/mod.rs

```diff
@@ -1143,6 +1153,35 @@ fn cmd_config_list(
     Ok(())
 }
 
+fn cmd_config_get(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    args: &ConfigGetArgs,
+) -> Result<(), CommandError> {
+    let name_path = args.name.split('.').collect_vec();
+    let values = command.resolved_config_values(&name_path)?;
+    match values.as_slice() {
```

Comment:
needs to filter out `is_overridden` entries. There may be multiple entries of the same name.